src/core/components/VTSocialLink.vue

```diff
@@ -22,7 +22,7 @@ const icons = {
   github: VTIconGitHub,
   linkedin: VTIconLinkedIn,
   slack: VTIconSlack,
-  twitter: VTIconTwitter,
+  twitter: VTIconX,
```

Comment:
For better flexibility to end users, I might prefer having both twitter and x props here. The twitter is still to the blue bird icon. The x is to the new one. So people can feel free to choose what they would like to use. What do you think?

Reply:
That also means we'd better keep the file `src/core/components/icons/VTIconTwitter.vue`.

Reply:
Good point. @4ndu-7h4k Would you mind fixing this?

Reply:
Sure @kiaking